soc/intel/intel_adsp/ace/power.c

```diff
@@ -43,7 +43,7 @@ __imr void power_init(void)
 #if CONFIG_SOC_INTEL_ACE15_MTPM
 	*((__sparse_force uint32_t *)sys_cache_cached_ptr_get(&adsp_pending_buffer)) =
 		INTEL_ADSP_ACE15_MAGIC_KEY;
-	cache_data_flush_range((__sparse_force void *)
+	sys_cache_data_flush_range((__sparse_force void *)
```

Comment:
Here too it would be helpful to explain why this has compiled before (i.e. what has changed so we need this now). 
It would seem cache.h has this:
include/zephyr/arch/cache.h:#define cache_data_flush_range(addr, size) arch_dcache_flush_range(addr, size

... so if the dcache line size is set, this should have worked. But sys_cache_data..() is better anyways, so no objections to the change itself.

Reply:
> Here too it would be helpful to explain why this has compiled before 

Yes please. Our CIs go well beyond compilation now so it's always very surprising to see a PR named "fix the build"!? Maybe it's an unusual configuration/corner case/toolchain? Is it important enough to be added to CI?